static/styles/zulip.css

```diff
@@ -436,9 +413,7 @@ a:hover code {
     opacity: 0.5;
 }
 
-#message_edit_tooltip:hover,
-#streams_inline_cog:hover,
-#streams_filter_icon:hover {
+#message_edit_tooltip:hover {
    opacity: 1.0;
 }
```

Comment:
A separate tooltip would be good, especially since it's not being used much at the moment and it could be more useful.